statsmodels/sandbox/gam_gsoc2015/gam_example.R

```diff
@@ -0,0 +1,47 @@
+## This files contains the R code to generate some test for the GAM function ##
+## Documentation for MGCV is available at: http://cran.r-project.org/web/packages/mgcv/mgcv.pdf
+
+library('pracma')
+library('mgcv')
+#library('gam')
+n = 100
+x = seq(from = -1, to = 1, length.out = n)
+
+funz = function(x){
+  2 * x * x * x - x + rnorm(n = n, mean = 0, sd = .1)
```

Comment:
Since you saved the data, y, x, with the results in the `.csv` file, we can just reload them.